in-adapter-api/src/test/java/gohigher/auth/support/JwtProviderTest.java

```diff
@@ -0,0 +1,140 @@
+package gohigher.auth.support;
+
+import static org.assertj.core.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.nio.charset.StandardCharsets;
+import java.util.Date;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import gohigher.global.exception.GoHigherException;
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.security.Keys;
+
+@DisplayName("JwtProvider 클래스의")
+class JwtProviderTest {
+
+	private static final long ACCESS_TOKEN_EXPIRE_LENGTH = 100000;
+	private static final int REFRESH_TOKEN_EXPIRE_LENGTH = 300000;
+	private static final int UNIT_TO_CONVERT_MILLI_TO_SECOND = 1000;
+	private static final String SECRET = "secretKey".repeat(6);
+
+	private final JwtProvider jwtProvider = new JwtProvider(SECRET, ACCESS_TOKEN_EXPIRE_LENGTH,
+		REFRESH_TOKEN_EXPIRE_LENGTH);
+
+	@DisplayName("createToken 메서드는")
+	@Nested
+	class createToken {
+
+		@DisplayName("value가 정해지면")
+		@Nested
+		class Value {
+			Long userId = 1L;
+			Date now = new Date();
```

Comment:
```suggestion

			private final Long userId = 1L;
			private final Date now = new Date();
```
아래 테스트에는 접근제어자와 final 이 포함되어 있는데 다른 이유가 있을까요??

Reply:
아니요! 클래스니까 다 붙이는 걸로 통일했습니다!